hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieParquetInputFormat.java

```diff
@@ -54,20 +45,16 @@
  */
 @UseRecordReaderFromInputFormat
 @UseFileSplitsFromInputFormat
-public class HoodieParquetInputFormat extends HoodieFileInputFormatBase implements Configurable {
+public class HoodieParquetInputFormat extends HoodieParquetInputFormatBase {
 
   private static final Logger LOG = LogManager.getLogger(HoodieParquetInputFormat.class);
 
-  // NOTE: We're only using {@code MapredParquetInputFormat} to compose vectorized
-  //       {@code RecordReader}
-  private final MapredParquetInputFormat mapredParquetInputFormat = new MapredParquetInputFormat();
-
-  protected HoodieDefaultTimeline filterInstantsTimeline(HoodieDefaultTimeline timeline) {
-    return HoodieInputFormatUtils.filterInstantsTimeline(timeline);
+  public HoodieParquetInputFormat() {
+    super(new HoodieCopyOnWriteTableInputFormat());
```

Comment:
why not 
```
this(new HoodieCopyOnWriteTableInputFormat())
```

Reply:
B/c there's no point in that indirection -- we can go straight to the super class